session-manager-android/build.gradle

```diff
@@ -23,11 +23,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
```

Comment:
pls move back to 1.8

Reply:
Have reverted this.

The version will still stay at 11, due to kotlin:
```
kotlinOptions {
        jvmTarget = '11'
    }
```